test/integration/services/data/test-insert-reduction.js

```diff
@@ -0,0 +1,38 @@
+const expect = require('chai').expect
+
+const insertReduction = require('../../../../app/services/data/insert-reduction')
+const Reduction = require('../../../../app/services/domain/reduction')
+const workloadCapacityHelper = require('../../../helpers/data/aggregated-data-helper')
+
+var reductionResult = {
+  table: 'reductions',
+  id: 0
+}
+
+var testReduction = new Reduction(1, 5, new Date(), new Date(), 'Test Note')
+var workloadOwnerId
+
+describe('/services/data/insert-reduction', function () {
+  before(function () {
+    return workloadCapacityHelper.getAnyExistingWorkloadOwnerId()
+      .then(function (id) {
+        workloadOwnerId = id
+        return workloadCapacityHelper.getAnyExistingReductionReasonId()
+          .then(function (id) {
+            testReduction.reasonForReductionId = id
+          })
+      })
+  })
+  it('should return an id when a valid reduction has been added', function () {
```

Comment:
Should add a test that checks that the reduction exists in the db.  Is there an existing data service you could use?

Reply:
That can be done when we have the query to get reductions completed. These tests can then be extended.